src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -292,67 +299,106 @@ public IEnumerable<string> EnumeratePSFiles()
 
         #region Private Methods
 
+        /// <summary>
+        /// Find PowerShell files recursively down from a given directory path.
+        /// Currently returns files in depth-first order.
+        /// Directory.GetFiles(folderPath, pattern, SearchOption.AllDirectories) would provide this,
+        /// but a cycle in the filesystem will cause that to enter an infinite loop.
```

Comment:
Has that been submitted as a bug to CoreFx?

Reply:
It's by design I think: https://docs.microsoft.com/es-es/dotnet/api/system.io.searchoption?view=netframework-4.7.2#remarks

Reply:
Doh!

Reply:
Dunno why that link was Spanish...